client/src/components/common/Button/ContainedButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  root: {
+    appearance: 'none',
+    background: 'hsl(22, 85%, 53%)',
+    borderRadius: '0.3rem',
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 2rem',
+    minWidth: '8rem',
+    '&:hover': {
+      backgroundColor: 'hsl(22, 85%, 45%)',
+    },
+  },
+};
```

Comment:
style should be added in separate file with `style.js` file then call it in `index.js` file for the button , and we need only one folder for the button, not 2 , just one & make it dynamic to accept man props so we can pass any thing we want .